Share with Gifski/Share_with_Gifski.entitlements

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+<plist version="1.0">
+<dict>
+	<key>com.apple.security.app-sandbox</key>
+	<true/>
+	<key>com.apple.security.application-groups</key>
+	<array>
+		<string>$(TeamIdentifierPrefix).gifski_video_share_group</string>
```

Comment:
```suggestion
		<string>$(TeamIdentifierPrefix).com.sindresorhus.Gifski.shareGroup</string>
```

?

Reply:
Apple says it should be `$(TeamIdentifierPrefix).<name>` for macOS apps. I can rename the group to `shareGroup` though if you prefer.

Reference: https://developer.apple.com/documentation/bundleresources/entitlements/com_apple_security_application-groups

Reply:
Ah, ok. Leave it as is then.